server.js

```diff
@@ -0,0 +1,14 @@
+const express = require("express");
+const app = express();
+
+
+//routes
+
+app.get('/ping', (req, res) => {
+    res.send("Hello devoob");
+})
+
+app.listen(4000, ()=> {
```

Comment:
Consider using environment variables for the port number instead of hardcoding it. This will make your application more flexible and easier to configure in different environments. [important]